lms/templates/ccx/enrollment.html

```diff
@@ -53,7 +54,7 @@ <h2 class="hd hd-2">${_("Batch Enrollment")}</h2>
   </div>
 
   <div>
-    <input type="submit" name="enrollment-button" class="enrollment-button" value="${_("Enroll")}">
+    <input type="submit" name="enrollment-button" class="enrollment-button" value="${_p("someone","Enroll")}">
     <input type="submit" name="enrollment-button" class="enrollment-button" value="${_("Unenroll")}">
```

Comment:
I would think that this `unenroll` also works with the same context. Maybe not enough to add it now, but we can make a comment on the final PR